lib/core/src/lib/form/components/widgets/date-time/date-time.widget.ts

```diff
@@ -42,68 +43,115 @@ import { WidgetComponent } from '../widget.component';
     ],
     templateUrl: './date-time.widget.html',
     styleUrls: ['./date-time.widget.scss'],
-    imports: [NgIf, TranslateModule, MatFormFieldModule, MatInputModule, MatDatetimepickerModule, FormsModule, ErrorWidgetComponent],
+    imports: [NgIf, TranslateModule, MatFormFieldModule, MatInputModule, MatDatetimepickerModule, ReactiveFormsModule, ErrorWidgetComponent],
     encapsulation: ViewEncapsulation.None
 })
-export class DateTimeWidgetComponent extends WidgetComponent implements OnInit {
+export class DateTimeWidgetComponent extends WidgetComponent implements OnInit, OnDestroy {
```

Comment:
This should be implemented as Angular custom form control with ControlValueAccessor , but I suppose it's not possible with the base WidgetComponent? 
ref. https://blog.angular-university.io/angular-custom-form-controls/

Reply:
As we discussed, we should refactor it more and remove extending WidgetComponent. 
Issue -> https://hyland.atlassian.net/browse/AAE-24760